### PR TITLE
[script.simkl] 2.2.0

### PR DESCRIPTION
**Pull request by ekleop**
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-scripts/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [script.foo.bar] 1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practise but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-scripts/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.

### CODE REVIEW

**Comment by ekleop** [comment]
Hi @Rechi  is there an approximate timeline when this will be merged?

File: script.simkl/addon.xml [hunk 1]
```diff
@@ -24,5 +24,9 @@
       <fanart>resources/fanart.jpg</fanart>
     </assets>
     <language>en</language>
+    <news>
```
Comment (enen92) [inline script.simkl/addon.xml]: the news tag is only available for krypton and later versions

File: script.simkl/crowdin.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+files:
```
Comment (enen92) [inline script.simkl/crowdin.yml]: please keep this file out of the submission

File: script.simkl/addon.xml (hunk 1 above)
Comment (enen92) [reply script.simkl/addon.xml]: Although I am ok with keeping it in the addon.xml since the tag will be ignored by jarvis

**Comment by enen92** [comment]
@ekleop please also drop all the .po files without translations. There's no point in having those since Kodi will always fallback to the en as the default lang

File: script.simkl/addon.xml (hunk 1 above)
Comment (ekleop) [reply script.simkl/addon.xml]: Doesn't krypton and later versions cannot install the addon? so they will use this tag or am I wrong?

Comment (Rechi) [reply script.simkl/addon.xml]: This add-on can be installed in Krypton and newer versions. Those versions do not have a `changelog.txt` fallback, so if you want to see a changelog inside Kodi, you have to keep the news tag.

**Comment by ekleop** [comment]
ok, I've removed yaml file + all empty language files.Everything should be fine now.

File: script.simkl/addon.xml (hunk 1 above)
Comment (ekleop) [reply script.simkl/addon.xml]: yeap, that is what I wanted to know. We left it specifically for this. Thanks.

**Comment by KarellenX** [comment]
@enen92 
Could you submit this if all is ok. It has been over 5 weeks since the PR was submitted.

Thanks

**Comment by KarellenX** [comment]
Thanks @enen92

**Pull request merged**